database/things/aiml.pldb

```diff
@@ -2,10 +2,10 @@ title AIML
 type dataNotation
 website http://www.aiml.foundation/
 documentation http://www.aiml.foundation/doc.html
+standsFor Artificial Intelligence Markup Language
 originCommunity https://mindsdb.com/community
 domainName aiml.foundation
-standsFor Artificial Intelligence Markup Language
 
-wikipedia https://en.wikipedia.org/wiki/Artificial_Intelligence_Markup_Language
+githubRepo https://github.com/AIML-Foundation
```

Comment:
can we move these changes to a separate commit?

and then just have one pull request about the tableSearch change?